### PR TITLE
Issue 62

### PR DESCRIPTION
**Pull request by Daniel-Houston**
### Description
Remove unused and unneeded fields from the PutMappingRequest.

Remove errant _field suffix from json properties
 
### Issues Resolved
https://github.com/opensearch-project/opensearch-java/issues/62
 
### Check List
- [ x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by Daniel-Houston** [comment]
This commit is a starting point, as this is auto-generated and I don't want to do any other code work until I know I'm on the right track. I've removed all 'body' fields besides 'properties' as that is the only field allowed in the body. I look forward to the conversation

File: java-client/src/main/java/org/opensearch/client/opensearch/_types/mapping/IndexField.java [hunk 1]
```diff
@@ -46,7 +46,7 @@
 
 import java.util.Objects;
 
-// typedef: _types.mapping.IndexField
+// typedef: _types.mapping.IndexSetting
```
Comment (dblock) [inline java-client/src/main/java/org/opensearch/client/opensearch/_types/mapping/IndexField.java:49]: How does this work? What does it do?

Comment (Daniel-Houston) [reply java-client/src/main/java/org/opensearch/client/opensearch/_types/mapping/IndexField.java]: This was done by mistake, I've reverted the change

**Comment by dblock** [comment]
@Daniel-Houston You need to amend your commit with `-s` for DCO check, please.

**Comment by mitalawachat** [comment]
DCO check seems to be failing: https://github.com/opensearch-project/opensearch-java/runs/4398721746?check_suite_focus=true
Also, maybe add unit-test / integ-test?

**Comment by dblock** [comment]
I know it's old, but maybe want to finish this @Daniel-Houston ?

**Comment by wbeckler** [comment]
@Daniel-Houston If there's anything about the DCO process that's giving you a hassle, can you raise it here and we can look into a process improvement or other way to unblock you?

**Comment by Daniel-Houston** [comment]
I'm working on this

**Comment by Daniel-Houston** [comment]
Things have changed fairly significantly since this I made this PR. Do you want me to rebase things onto the new main branch and submit a new CR?

The class has comments indicating it is auto-generated and manual edits made would be lost. Does my change even make sense given that condition? I submitted this PR to get a conversation going as I couldn't find the code which did the auto-generating. If you point me in the right direction I will make the changes where they need to happen

**Comment by Daniel-Houston** [comment]
Working on an updated CR into the new main branch

**Pull request closed without merging**